topaz-backend-main/main.py

```
@@ -130,50 +131,71 @@ def get_metrics(self):
+
+                if self.expected_iss and claims.get("iss") != self.expected_iss:
+                    return JSONResponse(status_code=401, content={"error": "Invalid token issuer"})
+
+                aud = claims.get("aud")
+                if aud and aud != "authenticated":
+                    return JSONResponse(status_code=401, content={"error": "Invalid token audience"})
+
+                # Attach user info
+                try:
+                    user_email = claims.get("email") or (claims.get("user_metadata") or {}).get("email")
+                except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)